organizers/2018-07-16.html

```diff
@@ -0,0 +1,134 @@
+    </ul>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/festicket.svg" style="width:40%" alt="Festicket" class="plain"></img>
+    <p>
+        <em>Europe's largest music festival platform, and UK's 6th fastest growing private tech start up.<br>Transforming the way music fans discover, book tickets and travel packages for festivals around the world.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/thread.png" style="width:10%" alt="Thread" class="plain"></img>
+    <p>
+        <em>We help people to feel happier and more self-confident, by making it easy to dress well, using stylists and data science. Dedicated to maintaining a great working environment, minimising politics and maximising impact.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/timeout.png" style="width:40%" alt="Time Out" class="plain"></img>
+    <p>
+        <em>The trusted global platform that inspires and enables people to experience the best of the city. Operating in 107 cities, across 39 countries, it has an average monthly global audience reach of over 111 million across all platforms.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/verve.svg" style="width:40%" alt="Verve" class="plain"></img>
+    <p>
+        <em>We turn word of mouth into your best sales channel.</em>
+    </p>
+</section>
+
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>📰 News</h2>
+    <h2>🎤 Talks</h2>
+    <h2>🍕 Food &amp; Drink</h2>
+</section>
+
+<section>
+    <h1>🐍 News</h1>
+    <p>Guido van Rossum steps down as Python's &ldquo;benevolent dictator for life&rdquo;.</p>
+</section>
+
+<section>
+    <h1>🇪🇺 EuroPython 🏴󠁧󠁢󠁳󠁣󠁴󠁿</h1>
+    <ul>
+        <li>July 23&ndash;29</li>
+        <li>Edinburgh, Scotland</li>
+        <li>Training passes sold out, but conference tickets are still available.</li>
+        <li><a href="http://ep2018.europython.eu">ep2018.europython.eu</a></li>
+    </ul>
+</section>
+
+<section>
+    <h1>🇬🇧 PyCon UK 🏴󠁧󠁢󠁷󠁬󠁳󠁿</h1>
+    <ul>
+        <li>September 15&ndash;19</li>
+        <li>Cardiff, Wales</li>
+        <li><a href="https://2018.pyconuk.org">2018.pyconuk.org</a></li>
+    </ul>
+</section>
+
+<section>
+    <h1>🎤 Talks</h1>
+    <p>
+        <strong>Ibukun Oluwayomi</strong> &mdash;<br>
+        &ldquo;Concurrency and Parallelism in Python: Not as Complicated as you Think&rdquo;
+    </p>
+    <p>
+        <strong>Mat Clayton</strong> &mdash;<br>
+        &ldquo;GraphQL and Django&rdquo;
+    </p>
+</section>
+
+<section>
+    <h1>🍕 Food &amp; Drink</h1>
+    <p>The Space Bar until ~20:45</p>
+    <p>Migrate to the Red Lion Pub (opposite)</p>
+</section>
+
+<section>
+    <h1>Want to give a talk? 🤔</h1>
+    <p>Please come and chat with us!</p>
+    <ul>
+        <li>Adam</li>
+        <li>Çağıl</li>
+        <li>Craig (that's me!)</li>
+        <li>David</li>
+        <li>James</li>
+        <li>Marco</li>
```

Comment:
I thought it would be a good idea to get us all to stand up at this point so that people know who we are. Listed alphabetically because why not.

Reply:
It is a good idea!